pysumma/pysumma/Decisions.py

```diff
@@ -0,0 +1,124 @@
+class Decisions:
+    def __init__(self, filepath):
+        # self.path = path
+        # self.filename = filename
+        self.filepath = filepath
+        self.simulStart = simul_datetime('simulStart', self.filepath)
+        self.simulFinsh = simul_datetime('simulFinsh', self.filepath)
+        self.soilCatTbl = option('soilCatTbl', self.filepath)
+        self.vegeParTbl = option('vegeParTbl', self.filepath)
+        self.soilStress = option('soilStress', self.filepath)
+        self.stomResist = option('stomResist', self.filepath)
+        self.num_method = option('num_method', self.filepath)
+        self.fDerivMeth = option('fDerivMeth', self.filepath)
+        self.LAI_method = option('LAI_method', self.filepath)
+        self.f_Richards = option('f_Richards', self.filepath)
+        self.groundwatr = option('groundwatr', self.filepath)
+        self.hc_profile = option('hc_profile', self.filepath)
+        self.bcUpprTdyn = option('bcUpprTdyn', self.filepath)
+        self.bcLowrTdyn = option('bcLowrTdyn', self.filepath)
+        self.bcUpprSoiH = option('bcUpprSoiH', self.filepath)
+        self.bcLowrSoiH = option('bcLowrSoiH', self.filepath)
+        self.veg_traits = option('veg_traits', self.filepath)
+        self.canopyEmis = option('canopyEmis', self.filepath)
+        self.snowIncept = option('snowIncept', self.filepath)
+        self.windPrfile = option('windPrfile', self.filepath)
+        self.astability = option('astability', self.filepath)
+        self.canopySrad = option('canopySrad', self.filepath)
+        self.alb_method = option('alb_method', self.filepath)
+        self.compaction = option('compaction', self.filepath)
+        self.snowLayers = option('snowLayers', self.filepath)
+        self.thCondSnow = option('thCondSnow', self.filepath)
+        self.thCondSoil = option('thCondSoil', self.filepath)
+        self.spatial_gw = option('spatial_gw', self.filepath)
+        self.subRouting = option('subRouting', self.filepath)
+
+class option:
+    def __init__(self, name, filepath):
+        self.filepath = filepath
+        self.open_read()
+        self.name = name
+        self.get_description()
+        self.options = self.get_options()
+
+    def open_read(self):
+        with open(self.filepath, 'rt') as f:
+            self.text = f.readlines()
+        return self.text
+
+    def get_line_no(self, text_startwith):
+        text = self.open_read()
```

Comment:
`option.open_read` is called upon instantiation, so we can just use the `self.text` variable in place of reading the file again.